node-setup/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM to-be-replaced-by-local-ref/base:ubuntu
+FROM to-be-replaced-by-local-ref/base:ubi-9.4-minimal
 
-RUN apt update && \
-    apt install -y --no-install-recommends xfsprogs mdadm && \
-    apt clean  && \
-    rm -rf /var/lib/apt/lists/*
+RUN microdnf update -y && \
+    microdnf install -y xfsprogs mdadm --nodocs --enablerepo almalinux-base-9 && \
```

Comment:
I recall mdadm had completely different behavior and command line parameters between versions. Just leaving a note in case e2e's around local disk setup would  failing after merging this.